deluca/envs/lung/_delay_lung.py

```diff
@@ -1,16 +1,4 @@
-# Copyright 2020 Google LLC
```

Comment:
Can you bring back the copyright notice at the top of the file

Reply:
ok will do